.github/workflows/update.yml

```
@@ -36,13 +36,15 @@ jobs:
         run: npm run sites:update
       - run: git status
       - name: commit changes to sites.md
-        run: |
-          git add SITES.md
-          git status
-          git commit --allow-empty -m "[Bot] Update SITES.md" -m "Committed by [iptv-bot](https://github.com/apps/iptv-bot) via [update](https://github.com/iptv-org/epg/actions/runs/${{ github.run_id }}) workflow." --no-verify
-      - name: push all changes to the repository
         if: ${{ !env.ACT && github.ref == 'refs/heads/master' }}
```

Can we add this check to an existing `if` statement?

```suggestion
        if: ${{ !env.ACT && github.ref == 'refs/heads/master' && [ -n "$(git diff --exit-code --name-only SITES.md)" ] }}
```

---

I'm not sure if that's a valid action expression. It is safer to check the condition on shell.